src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -375,11 +370,11 @@ export class NotebookService extends Disposable implements INotebookService {
 	private sendNotebookCloseToProvider(editor: INotebookEditor): void {
 		let notebookUri = editor.notebookParams.notebookUri;
 		let uriString = notebookUri.toString();
-		let manager = this._managersMap.get(uriString);
-		if (manager) {
+		let managers = this._managersMap.get(uriString);
```

Comment:
manager was a minnomer becasue at every entry in _managersMap we store an array or managers as value and not a single manager. So fixing the name makes the code read better.